config.example/group_vars/all.yml

```diff
@@ -30,6 +30,7 @@
 
 # NTP configuration
 # Playbook: chrony-client
+install_chrony: yes
```

Comment:
can we call this something like `chrony_install` so it matches the other vars?

Reply:
Yeah good call